src/utils/helpers/with-unique-name/__snapshots__/with-unique-name.spec.js.snap

```diff
@@ -1,7 +1,7 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`withUniqueName HOC matches the snapshot when the it is passed an anonymous component with no name or display name 1`] = `
-<WithUniqueName(Component)
+<WithUniqueName(Unknown)
```

Comment:
Does the `Unknown` is correct?

Reply:
Yes, this is intentional - when the HOC wraps a component without a display name.